server/src/utils/request.utils.ts

```diff
@@ -0,0 +1,50 @@
+import axios from 'axios';
+import { serialize } from 'bson';
+import express, { Request, Response, Application } from 'express';
+import { ModuleConfig } from '../classes/moduleConfig';
+
+type Handler = (req: any, res: Response) => Promise<void>;
+type Resolver = (body: any) => Promise<string[] | Error>;
+type Serializer = (data: Object) => Object | Buffer;
+type PicturePromisePayload = { paths: string[]; config: ModuleConfig };
+
+const getAbstractHandler: (serializer?: Serializer) => (resolver: Resolver) => Handler =
```

Comment:
I would rename `getAbstractHandler` to "getHandler" and remove the current `getHandler`. I don't think we will add any more handlers, even the current ones could be removed.